lib/csu/common/crtbrand.S

```diff
@@ -47,3 +47,12 @@ __FBSDID("$FreeBSD$");
 2:	.p2align	2
 3:	.4byte		__FreeBSD_version
 4:
+	.section .note.tag,"aG",%note,.freebsd.noteG,comdat
```

Comment:
Shouldn't this be conditional on `__CheriBSD__` if we're going to all the trouble of defining it? Otherwise `__CheriBSD_version` won't be defined for -DWITHOUT_CHERIBSD, and even if it were you wouldn't want to include the note.

Reply:
Yes, I wanted to leave it out of this review to wait for the final decision on this. Since you mentioned it, I added the ifdef.